### ISSUE TITLE
Write standard complete dummy data for local database testing

### ISSUE THREAD

**Issue by King-Mob** [DigitalCommons/land-explorer-front-end#112]
**Description**

Currently we manage our databases using migration and seeding scripts that are repeatable, so that we can be assured that the state of a database on local is the same as on staging and the same on production.

It would be easier to write these scripts if we could drop the local database whenever we wanted and rebuild it from scratch, knowing that nothing will be missing when we want to start testing features again.

Basically, make testing migration scripts easier and quicker so we can do more of it, and be more relaxed about running migration scripts in production.

**Acceptance Criteria**

Any branch of the back-end has full seeding scripts that when run generate all the data required for the features in that branch.

This includes a test user, a data group, items in those data groups, item types etc.

**Comment by lin-d-hop** [comment]
Looks like dev test tyep work. 

@King-Mob will you close this when you are happy it is done.

**Comment by King-Mob** [comment]
Happy that it's complete for now.

As we add more back-end features we will probably need to add to the dummy data so that it's easy to test.
